config.yaml

```diff
@@ -1,9 +1,9 @@
 log_level: debug
 store_kind: fleetscheduler
-facility_code: fr9
+facility_code: sandbox
 fleetdb_api:
-  endpoint: http://192.168.1.191:8000
-  disable_oauth: true
+  endpoint: http://localhost:8000
+  disable_oauth: false
```

Comment:
probably want this as true for `sandbox`

Reply:
thats me getting mixed up. Thanks